<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>eBui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,15,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>19</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>116</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Open">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-folder-50.png</normaloff>:/Icons/Icons/icons8-folder-50.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="New">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-add-50.png</normaloff>:/Icons/Icons/icons8-add-50.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Simulate">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-electronics-50.png</normaloff>:/Icons/Icons/icons8-electronics-50.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Wire">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-wire-50.png</normaloff>:/Icons/Icons/icons8-wire-50.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-save-50.png</normaloff>:/Icons/Icons/icons8-save-50.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="AddComponent">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/Icons/Icons/icons8-heater-resistor-50.png</normaloff>:/Icons/Icons/icons8-heater-resistor-50.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>My Components</string>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/Icons/icons8-folder-50.png</normaloff>:/Icons/Icons/icons8-folder-50.png</iconset>
          </property>
         </column>
         <item>
          <property name="text">
           <string>-&gt;Voltage Source</string>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/Icons/Icons/Voltage.png</normaloff>:/Icons/Icons/Voltage.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x:0     y:0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
